app/webapp/templates/feed.tmpl

```diff
@@ -2,56 +2,179 @@
+                font-weight: bold;
+                display: block;
+                font-size: 1.25rem;
+                white-space: nowrap;
+                padding-bottom: 0.125em;
+            }
+            .ump-feed-master-info {
+                font-size: 1rem;
+            }
+            .ump-feed-master-header-link {
+                color: inherit;
+                text-decoration: none;
+                display: inline-block;
+                padding: 4px 0;
+                border-bottom: 1px dotted #999;
+            }
+            .ump-feed-master__data-row {
+                display: flex;
+                padding: 1rem;
+                font-size: 1rem;
+                border-bottom: 1px solid rgba(0, 0, 0, 0.3);
+            }
+            .ump-feed-master__data-row-player-cell {
+                width: 38px;
+                text-align: center;
+            }
+            @media (min-width: 760px) {
+                .ump-feed-master__data-row-player-cell {
+                    width: 58px;
+                }
+            }
+            .ump-feed-master__data-row-info-cell {
+                flex: 1;
+                padding-left: 0.75rem;
+            }
+            .ump-feed-master-timestamp-cell {
+                margin-left: auto;
+                color: rgba(0, 0, 0, 0.35);
+                padding-top: 0.5rem;
+                font-size: .825rem;
+            }
+            @media (min-width: 1240px) {
+                .ump-feed-master__data-row-info-cell {
+                    display: flex;
+                }
+                .ump-feed-master-timestamp-cell {
+                    padding-top: 0;
+                    font-size: 1rem;
+                }
+            }
+            .ump-feed-master-footer {
+                font-size: .825rem;
+                text-align: center;
+                padding: 1rem;
+                background: rgba(0, 0, 0, 0.2);
+            }
+            .fa-volume-up {
+                color: inherit;
+            }
+        </style>
+
 </head>
 
 
 <body>
 
-    <nav class="navbar navbar-light fixed-top">
-        <div class="navbar-brand" href="#">
-            <img src="/static/podcast.png" width="30" height="30" alt="">
-            Feed Master <span id="name"><a href="{{.Link}}">{{.Name}}</a> - {{.Description}}</span>
+
+<header class="ump-feed-master-header">
+    <div class="ump-feed-master-header__brand">
+        <div>
+            <img src="/static/podcast.png" class="ump-feed-master-logo" alt="feed master logo">
+        </div>
+        <div>
+            <span class="ump-feed-master-name">Feed Master</span>
+            <span class="ump-feed-master-info">{{.Description}} <a href="{{.Link}}" class="ump-feed-master-header-link">«{{.Name}}»</a></span>
+        </div>
+    </div>
+    <div class="ump-feed-master-header__meta">
+        {{len .Items}} feeds, {{.LastUpdate.Format "02 Jan 2006 15:04:05 MST"}}
+    </div>
+</header>
+
+<main class="ump-feed-master">
+
```

Comment:
missing {{range .Items}}

Reply:
@svacad  thank you!

Reply:
@umputun I restored missed template part.